pkg/fleet/internal/cdn/cdn_regular.go

```diff
@@ -20,12 +20,12 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
-type cdnRemote struct {
+type cdnRegular struct {
```

Comment:
Can't you call them 
- CDNNoOp
- CDN
- CDNRCClient

Reply:
I don't have a strong opinion. Done in [65055a5](https://github.com/DataDog/datadog-agent/pull/30369/commits/65055a58be1c564020787e82bc140bcf2c8df94d)